apps/web/components/dashboard/bookmarks/EditorCard.tsx

```diff
@@ -150,6 +177,50 @@ export default function EditorCard({ className }: { className?: string }) {
               : "Press ⌘ + Enter to Save"
             : "Save"}
         </ActionButton>
+
+        <MultipleChoiceDialog
+          open={open}
+          setOpen={setOpen}
+          title={`Import URLs as separate Bookmarks?`}
+          description={`The input contains multiple URLs on separate lines. Do you want to import them as separate bookmarks?`}
+          onClose={() => {
+            setImportData(initialImportData);
+          }}
+          actionButtons={[
+            () => (
+              <ActionButton
+                type="button"
+                variant="secondary"
+                loading={isPending}
+                onClick={() => {
+                  mutate({ type: "text", text: importData.text });
+                  setOpen(false);
+                  setImportData(initialImportData);
+                }}
+              >
+                Import as Text Bookmark
+              </ActionButton>
+            ),
+            () => (
+              <ActionButton
+                type="button"
+                variant="destructive"
+                loading={isPending}
+                onClick={() => {
+                  importData.urls.forEach((url) =>
+                    mutate({ type: "link", url: url.toString() }),
+                  );
+                  setOpen(false);
+                  setImportData(initialImportData);
+                }}
+              >
+                Import as separate Bookmarks
+              </ActionButton>
+            ),
+          ]}
+        >
+          <div></div>
```

Comment:
Triggers are optional in dialogs. So if you modify `MultipleChoiceDialog` to have an optional `children` and only create the trigger node if `children` is set, you won't need this.